tools/CreateSolution.ps1

```diff
@@ -0,0 +1,5 @@
+dotnet new sln;
```

Comment:
?

Reply:
Go into any directory and call this script, you would get solution with all the projects.

Reply:
Sure, but why is it here